index.html

```diff
@@ -0,0 +1,141 @@
+            <li class=""><a href="#" class="nav-link">Programs</a></li>
+            <li class=""><a href="#" class="nav-link">Join</a></li>
+            <li class=""><a href="#" class="nav-link">Sponsor</a></li>
+            <li class=""><a href="#" class="nav-link">News</a></li>
+            <li class="cc"><a href="https://www.who.int/">WHO</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <section class="home-section1">
+      <h6 id="h6">"Hello! Together with WHO"</h6>
+      <h1 id="h1">
+        WORLD HEALTH <br />
+        SUMMIT 2023
+      </h1>
+      <div class="test-icon"></div>
+      <div class="home-section-div1">
+        <p class="home-section-p1">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Germany.</p>
+      </div>
+      <i id="dollar" class="fa fa-usd" aria-hidden="true"></i>
+      <i id="yen" class="fa fa-jpy" aria-hidden="true"></i>
+      <i id="cross" class="fa fa-crosshairs" aria-hidden="true"></i>
+      <h3 id="h3">2023.10.15(THU) ~ 17(FRI)</h3>
+      <p class="home-section-p2">@ JW Marriott Hotel, Berlin, Germany</p>
+    </section>
+    <section class="home-section2">
+      <h5 id="h5">
+        Main Program
+        <hr class="summit-partner-line" />
+      </h5>
+      <div class="home-section-main-div">
+        <div class="home-section2-div1">
+          <img src="./icons/speech1.png" alt="" />
+          <h3>Lecture</h3>
+          <p class="section2-p">Listen to the speakers from various countris about the messages of sharing and opening.</p>
+        </div>
+        <div class="home-section2-div1">
+          <img class="lecture" src="./icons/ticket-office.png" alt="" />
+          <h3>CC Exhibition</h3>
+          <p class="section2-p">Listen to the speakers from various countris about the messages of sharing and opening.</p>
+        </div>
+        <div class="home-section2-div1">
+          <img src="./icons/chat.png" alt="" />
+          <h3>Forum</h3>
+          <p class="section2-p">Listen to the speakers from various countris about the messages of sharing and opening.</p>
+        </div>
+        <div class="home-section2-div1">
+          <img src="./icons/workshop (1).png" alt="" />
+          <h3>Workshop</h3>
+          <p class="section2-p">Listen to the speakers from various countris about the messages of sharing and opening.</p>
+        </div>
+        <div class="home-section2-div1">
+          <img src="./icons/workshop.png" alt="" />
+          <h3>CC Ignite</h3>
+          <p class="section2-p">Listen to the speakers from various countris about the messages of sharing and opening.</p>
+        </div>
+      </div>
+      <div class="button-div">
+        <button class="button1" type="button">Join World Health Summit 2023</button>
+      </div>
+      <div class="desktop-button"><a href="">SEE THE WHOLE PROGRAM</a></div>
```

Comment:
- [ ] Great work on the layout of your page! :+1: However, please add more space between each card to match the template. As it currently stands, there is not enough spacing between the cards, which can make it difficult for users to distinguish between them.
By adding more space between each card, you can improve the visual hierarchy of the page and make it easier for users to navigate through the content. This will also help to ensure that the design remains consistent with the template, which is important for maintaining the overall aesthetic of the website.

**Your design**

![Screenshot from 2023-03-16 07-29-34](https://user-images.githubusercontent.com/105117832/225618915-68b63f54-29c9-47c6-97cc-65d4a8f318ac.png)

**Template**

![Screenshot from 2023-03-16 07-30-47](https://user-images.githubusercontent.com/105117832/225618991-65fa35ea-f0dc-416f-9306-8013fabf062a.png)


- [ ] You have done an excellent job in following the semantics within your project :+1:. However, please keep in mind that elements like the five main courses, such as 'Lecture' and 'CC Exhibition' should be enclosed within an `article` tag. This is because these elements are reusable and contain a significant amount of information.
By using the `article` tag, you will be indicating that these elements are standalone and can be reused on other pages within the website. It also enhances accessibility by allowing screen readers to understand the content more clearly, thereby improving the overall user experience.